### ISSUE TITLE
Hide external or health system workqueue

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#6103]
**Bug description:**
Hide external or health system workqueue. Since there is only one hospital for Niue, we wont require External workqueue in in progress of RA/ Registrar. So need to have a system to hide this workqueue.

**Steps to reproduce:**
1. Login to RA/ Registrar
2. Click in Progress>External

**Actual result:**
Shows external or health system workqueue where records fetched by API is shown. But Niue don't require this.

**Expected result:**
Hide external or health system workqueue

**Screen recording:**
![Hide external or health system workqueue](https://github.com/opencrvs/opencrvs-core/assets/94350336/08329131-d7f9-4970-bf74-471fbbf0cf76)

**Tested on:**
https://login.niue.opencrvs.org/?lang=en

**Comment by euanmillar** [comment]
@SyedaAfrida @jpye-finch  I dont think this is a bug.  The external workqueue cant be hidden and may be used in future for external notifications from things like public portals or perhaps used for NZ integrations.

**Comment by jpye-finch** [comment]
Would be fixed with #7073
